internal/handlers/handlers.go

```diff
@@ -1 +1,77 @@
 package handlers
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+func Index(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "text/html; charset=utf-8")
+	http.ServeFile(w, r, "index.html")
+}
+
+func Upload(w http.ResponseWriter, r *http.Request) {
+	if err := r.ParseMultipartForm(10 << 20); err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
```

Comment:
Это клиент что-то не так сделал

```suggestion
		http.Error(w, err.Error(), http.StatusBadRequest)
```